packages/live-share/src/AzureLiveShareHost.ts

```diff
@@ -0,0 +1,208 @@
+import { AzureMember, IAzureAudience } from "@fluidframework/azure-client";
+import {
+    IClientInfo,
+    IFluidContainerInfo,
+    IFluidTenantInfo,
+    ILiveShareHost,
+    INtpTimeInfo,
+    UserMeetingRole,
+} from "./interfaces";
+import { Deferred } from "./internals";
+
+/**
+ * @alpha
+ * This host enables using Live Share through your own `AzureClient` implementation.
+ * This is not intended to be used with `LiveShareClient`.
+ * We provide no SLA guarantees on this implementation while it is in alpha.
+ * 
+ * @remarks
+ * To use this API, first pass your `ContainerSchema` through the `proxyLiveDataObjectClass` function.
```

Comment:
getLiveShareContainerSchemaProxy instead of proxyLiveDataObjectClass?

Reply:
oops good catch